shopify/wishlist-template/web/routes/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function () {
     submit,
     formState: { isDirty },
   } = useActionForm(api.shopifyShop.update, {
-    findBy: shop.id,
+    findBy: shop?.id,
```

Comment:
I get an error when I try to interact with this form (save an updated frequency)

![CleanShot 2024-11-22 at 09 15 16@2x](https://github.com/user-attachments/assets/10707071-089c-4ef3-85ff-cb480528b1ab)

Reply:
I think this is linked to the submit

Reply:
Nope, I fixed the issue. I was missing id: true in the select